--- src/SomeAntics/SomeAntics/Words/Functional/YouTheWord.cs
+++ src/Antics.SomeLoader/SomeAntics/Words/Functional/YouTheWord.cs
@@ -38,7 +_,7 @@
 		}
 
 		public override void OnTransferred(Interactable before, Interactable after) {
-			base.OnTransferred(before, after);
+			base.do_OnTransferred(before, after);
 			before.Velocity = Vector2.Zero;
 			after.Level.MoveToEnd(after.Level.IndexOf(after));
 		}
