--- src/SomeAntics/SomeAntics/Particles/SoftHomingParticle.cs
+++ src/Antics.SomeLoader/SomeAntics/Particles/SoftHomingParticle.cs
@@ -1,17 +_,23 @@
 using Microsoft.Xna.Framework;
+using SomeAntics.API.Core.Types;
 using SomeAntics.Objects.Interactables;
 
 namespace SomeAntics.Particles
 {
-	public class SoftHomingParticle : Particle
+	public class SoftHomingParticle : ModParticle
 	{
 		public Interactable Target {
 			get;
 			set;
 		}
 
-		public SoftHomingParticle(Vector2 position, Color color, Interactable target, int size, int lifespan)
+		internal SoftHomingParticle(Vector2 position, Color color, Interactable target, int size, int lifespan)
-			: base(position, color, size, lifespan) {
+			: base(position, color, size, lifespan, "Antics") {
+			Target = target;
+		}
+
+		public SoftHomingParticle(Vector2 position, Color color, Interactable target, int size, int lifespan, string modName) 
+            : base(position, color, size, lifespan, modName) {
 			Target = target;
 		}
 
