--- src/SomeAntics/SomeAntics/SomeAntics.csproj
+++ src/Antics.SomeLoader/SomeAntics/SomeAntics.csproj
@@ -1,22 +_,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../Configuration.targets" />
   <PropertyGroup>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Library</OutputType>
     <TargetFramework>net5.0</TargetFramework>
     <Version>1.0.0.0</Version>
     <Company>SomeAntics</Company>
     <RootNamespace>SomeAntics</RootNamespace>
+    <Platforms>AnyCPU</Platforms>
   </PropertyGroup>
   <PropertyGroup Condition="$(Configuration.Contains('Server'))">
-    <OutputType>Exe</OutputType>
+    <OutputType>Library</OutputType>
     <OutputName>$(OutputName)Server</OutputName>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+  </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="Icon.bmp" />
     <EmbeddedResource Include="Icon.ico" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="MonoGame.Framework.DesktopGL" Version="3.8.0.1641" />
+    <PackageReference Include="MonoMod" Version="21.4.29.1" />
+    <PackageReference Include="MonoMod.Common" Version="21.6.13.1" />
+    <PackageReference Include="MonoMod.RuntimeDetour" Version="21.4.29.1" />
+    <PackageReference Include="MonoMod.Utils" Version="21.4.29.1" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="WebmiliosCommons">
