--- src/SomeAntics/SomeAntics/Words/FunctionalWord.cs
+++ src/Antics.SomeLoader/SomeAntics/Words/FunctionalWord.cs
@@ -1,10 +_,11 @@
 using Microsoft.Xna.Framework;
+using SomeAntics.API.ModTypes;
 using SomeAntics.Objects.Interactables;
 using WordLibrary;
 
 namespace SomeAntics.Words
 {
-	public abstract class FunctionalWord : Word
+	public abstract partial class FunctionalWord : Word
 	{
 		public static readonly Color[] defaultColors = new Color[2] {
 			Color.Black,
@@ -18,6 +_,7 @@
 
 		protected FunctionalWord(string text)
 			: base(text) {
+			_text = GlobalFunctionalWord.GetModifiedWord(text);
 		}
 
 		public virtual void Affect(SomeAntics game, Interactable source) {
